src/wp-includes/functions.php

```diff
@@ -1882,7 +1882,8 @@ function wp_nonce_field( $action = -1, $name = '_wpnonce', $referer = true, $ech
  * @return string Referer field HTML markup.
  */
 function wp_referer_field( $echo = true ) {
-	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( wp_unslash( $_SERVER['REQUEST_URI'] ) ) . '" />';
+	$request_url = remove_query_arg( '_wp_http_referer', wp_unslash( $_SERVER['REQUEST_URI'] ) );
+	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( $request_url ) . '" />';
```

Comment:
Align the equal sign.

```suggestion
	$request_url   = remove_query_arg( '_wp_http_referer', wp_unslash( $_SERVER['REQUEST_URI'] ) );
	$referer_field = '<input type="hidden" name="_wp_http_referer" value="' . esc_attr( $request_url ) . '" />';
```